### DISCUSSION TITLE
Update Tardis Chunk Generator on CurseForge

### DISCUSSION THREAD

**Discussion by Walker50364** [eccentricdevotion/TARDIS#717] [General]
Good evening using an Aternos server, the available version of the Tardis Chunk Generator plugin is the one on the curseforge site but in version 1.12, can you update the version on the site so that on Aternos, version 1.19 is compatible and installable.

plugin link : https://www.curseforge.com/minecraft/bukkit-plugins/tardischunkgenerator

Sincerely, Walker50364

**Comment by eccentricdevotion** [comment]
TARDIS and TARDISChunkGenerator uploaded to Curseforge (awaiting approval).
The files are for 1.19.4 - unlike the version shown on the page - Curseforge does not allow setting the version to 1.19.4...

**Reply by Walker50364** [reply]
Thanks waiting for the approval.
